.github/workflows/test.yaml

```diff
@@ -0,0 +1,13 @@
+name: test
+on: [push, pull_request]
```

Comment:
est-ce qu'on ne devrait pas les mettres pour les `pull_request` seulement ? 🤔 
Si on regarde "Checks" (donc les actions déroulées), on peut voir qu'on a 2 fois "tests". 
Sais-tu si c'est deux instances de tests différentes ou si c'est une instance reportée 2 fois ?

Reply:
Difficile à dire: je pense qu'il y a le test pour le push et le test pour la pull request. Donc il s'agit du même commit et comme j'ai indiqué "on: [push, pull_request]" je pense que c'est deux instances de tests différentes mais j'ai aucune certitude...

 Idéalement on devrait pas faire jouer les tests à chaque push? De toutes façons pour pouvoir faire une pull request il faut faire un push... Tu penses que c'est too much de tester systématiquement avant de créer la PR?

Reply:
Je ne sais pas, c'est la question que je me pose 😋  Si c'est la même instance de test, c’est pas dérangeant, c'est que de l'affichage. Sinon c'est plus chiant. Bon, on ne paie pas, mais pour le principe ^^

Reply:
Ha bah oui c'est vrai que ça fait partie des choses qui peuvent être facturées... Je vais me renseigner pour le principe du coup, ça peut toujours être utile de savoir ^^

Reply:
Si j'en crois la doc sur https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions
```
# Triggers the workflow on push or pull request events
on: [push, pull_request]
```
Il semblerait que se soit l'un ou l'autre. Donc ça évoque une seule instance. Mais le fait que ça apparaisse deux fois n'est pas tellement convaincant...

Reply:
Je propose qu'on les lancent seulement lorsqu'une pull request est ouverte du coup 😉

Reply:
Ok ça marche je modifie ça. En ce qui concerne l'action de `deploy`, comment veux-tu t'organiser pour qu'on puisse le faire ensemble?

Reply:
L'action deploy tu pourras surement juste lier vercel par l'interface et créer un `vercel.json` dans le projet